dockstore-integration-testing/src/test/java/io/dockstore/webservice/languages/SnakemakePluginIT.java

```diff
@@ -0,0 +1,160 @@
+            final String absolutePath = temporaryTestingPlugins.toFile().getAbsolutePath();
+            System.out.println("path for support: " + absolutePath);
+            SUPPORT = new DropwizardTestSupport<>(DockstoreWebserviceApplication.class, DROPWIZARD_CONFIGURATION_FILE_PATH,
+                ConfigOverride.config("languagePluginLocation", absolutePath));
+        } catch (IOException e) {
+            throw new RuntimeException("could not create temporary directory");
+        }
+    }
+
+    private static TestingPostgres testingPostgres;
+
+    @SystemStub
+    public final SystemOut systemOut = new SystemOut();
+    @SystemStub
+    public final SystemErr systemErr = new SystemErr();
+
+
+    @BeforeAll
+    public static void dropAndRecreateDB() throws Exception {
+        CommonTestUtilities.dropAndRecreateNoTestData(SUPPORT);
+        SUPPORT.before();
+        testingPostgres = new TestingPostgres(SUPPORT);
+    }
+
+    @AfterAll
+    public static void afterClass() {
+        SUPPORT.getEnvironment().healthChecks().shutdown();
+        SUPPORT.after();
+    }
+
+    @BeforeEach
+    public void setup() throws Exception {
+        DockstoreWebserviceApplication application = SUPPORT.getApplication();
+
+        // used to allow us to use tokenDAO outside the web service
+        Session session = application.getHibernate().getSessionFactory().openSession();
+        ManagedSessionContext.bind(session);
+
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+    }
+
+    @Test
+    void testSnakemakeLanguagePlugin() {
+        MetadataApi metadataApi = new MetadataApi(getWebClient(false, "n/a", testingPostgres));
+        final List<DescriptorLanguageBean> descriptorLanguages = metadataApi.getDescriptorLanguages();
+        // should have default languages plus galaxy via plugin
+        assertTrue(descriptorLanguages.stream().anyMatch(lang -> lang.getFriendlyName().equals(DescriptorLanguage.CWL.getFriendlyName())));
+        assertTrue(descriptorLanguages.stream().anyMatch(lang -> lang.getFriendlyName().equals(DescriptorLanguage.WDL.getFriendlyName())));
+        assertTrue(descriptorLanguages.stream().anyMatch(lang -> lang.getFriendlyName().equals(DescriptorLanguage.NEXTFLOW.getFriendlyName())));
+        assertTrue(descriptorLanguages.stream().anyMatch(lang -> lang.getFriendlyName().equals(DescriptorLanguage.SMK.getFriendlyName())));
+        // should not be present
+        assertFalse(descriptorLanguages.stream().anyMatch(lang -> lang.getFriendlyName().equals(DescriptorLanguage.SWL.getFriendlyName())));
+    }
+
+    @Test
+    void testSnakemakePluginPublish() {
+        final ApiClient webClient = BaseIT.getWebClient(TestUser.TEST_USER2.dockstoreUserName, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(webClient);
+        final io.dockstore.openapi.client.ApiClient openAPIClient = BaseIT.getOpenAPIWebClient(TestUser.TEST_USER2.dockstoreUserName, testingPostgres);
```

Comment:
I see a mix of openapi and swagger models. Intentional? If not, might be best to stick with openapi since this is a new file